File: agent/verticals/legal_services.py
"""Legal and professional services vertical definition."""

from agent.verticals.base import VerticalDefinition, tabs

VERTICAL = VerticalDefinition(
    key="legal_services",
    label="Legal Services",
    risk_level="high",
    entity_label_singular="service",
    entity_label_plural="services",
    default_plan_label="Legal services plan",
    crm_tabs=tabs(
        ("overview", "Overview"),
        ("readiness", "Readiness"),
        ("catalog", "Services"),
        ("documents", "Documents"),
        ("leads", "Intake/Leads"),
        ("compliance", "Compliance"),
        ("activity", "Conversations"),
        ("prompt", "Prompt"),
        ("controls", "Controls"),
    ),
    entity_types=("legal_service", "lawyer", "practice_area", "document_template", "jurisdiction"),
    readiness_checks=("services", "attorneys", "jurisdictions", "consultation", "disclaimers", "pricing"),
    action_types=("SHOW_ENTITIES", "START_INTAKE", "REQUEST_CONSULTATION", "CAPTURE_LEAD", "HANDOFF_TO_LAWYER"),
)

File: crm/src/verticals/legalServices.ts
import { tab } from './shared';
import type { CrmVerticalDefinition } from './types';

export const legalServicesVertical: CrmVerticalDefinition = {
  key: 'legal_services',
  label: 'Legal Services',
  riskLevel: 'high',
  entityLabelSingular: 'service',
  entityLabelPlural: 'services',
  defaultPlanLabel: 'Legal services plan',
  clientTabs: [
    tab('overview', 'Overview'),
    tab('readiness', 'Readiness'),
    tab('catalog', 'Services'),
    tab('documents', 'Documents'),
    tab('leads', 'Intake/Leads'),
    tab('compliance', 'Compliance'),
    tab('activity', 'Conversations'),
    tab('prompt', 'Prompt'),
    tab('controls', 'Controls'),
  ],
  entityTypes: ['legal_service', 'lawyer', 'practice_area', 'document_template', 'jurisdiction'],
  readinessChecks: ['services', 'attorneys', 'jurisdictions', 'consultation', 'disclaimers', 'pricing'],
};
